content/millennium/dstu2/infrastructure/document-reference.md

```diff
@@ -144,8 +148,9 @@ Create new documents. Currently limited to unstructured clinical notes or docume
 
 _Implementation Notes_
 
-* The modifier elements [implicitRules], [modifierExtension] and [relatesTo] are not supported and will be rejected if present.
-* Currently only XHTML formatted documents are supported. You can validate your document using any available strict XHTML 1.0 validator (eg: [w3 validator] or this [html5 validator]).
+- The modifier elements [`implicitRules`], [`modifierExtension`], and [`relatesTo`] are not supported and will be rejected if present.
```

Comment:
Figured I'd check both DSTU2 and R4 resources' behavior here.

- DSTU2 resource is throwing generic 500s for all three 
  - Looks like an "enhancement opportunity" on the DSTU2 resource if we want to expose better details to developers about why their request may have failed.
- R4 resource throws 422s for implicitRules and modifierExtension existing at all, and various errors for relatesTo depending on how it's populated / not.

Reply:
We decided to remove the unnecessary "extra" sections around these elements throughout the document - they aren't required since we already have clear language around which fields are supported for a POST action to create the document, and we don't need to pull these items out specifically as not supported.

We should probably be throwing 422's for all of these elements, since they're not supported, but right now we throw the generic 500 on the DSTU2 resource.  I'm going to work on getting a request to revisit how the HTTP codes kick out here.